### DISCUSSION TITLE
How to dynamic loading if I use PbrBundle ?(How to drop PbrBundle to Improve performance)

### DISCUSSION THREAD

**Discussion by zzhgithub** [bevyengine/bevy#5655] [Q&A] [answered]
Hi:

I want to build a micraft like game; and when i render a face of  block like this:
```rust
commands.spawn_bundle(PbrBundle {
        mesh: quad_handle.clone(),
        material: texture.forward.clone(),
        transform: get_transform_by_face_type_form_vec3(FaceType::Forward, vec3, cube_size),
        ..default()
    });
```

i can see it  forever~ but i just to render it when i close to it. how can i do it?(how to drop pbrbundle?)
Maybe it not the best method. it you have a better one .please tell me

Thanks!

**Comment by wilk10** [comment, marked as answer]
You can toggle the 'is_visible' bool in the 'Visibility' component that is part of the 'PbrBundle'

**Reply by zzhgithub** [reply]
thank an i find that  i can cache entity to use despawn to drop an entity.
thank you.

**Comment by zzhgithub** [comment]
For help anyone else if make a demo https://github.com/zzhgithub/game-test/tree/feat01
and igrone the chinese comment .haha
